Modules/Filtering/ImageGrid/wrapping/test/OrientImageFilterTest.py

```diff
@@ -27,7 +27,7 @@
 reoriented_image = itk.orient_image_filter(
     image,
     use_image_direction=True,
-    desired_coordinate_orientation=itk.SpatialOrientationEnums.ValidCoordinateOrientations_ITK_COORDINATE_ORIENTATION_LPI
+    desired_coordinate_orientation=itk.AnatomicalOrientation(itk.AnatomicalOrientation.PositiveEnum_RAS)
```

Comment:
@thewtex It does not appear the default Python wrapping add the implicit conversions to the AnatomicalOrientation. Can I leave into you to add the SWIG "sugar" to make the interface cleaner in Python if desired?

Reply:
@blowekamp what do you think about an explicitly overloaded method that accepts the new type?

Reply:
The "new type"? Please be more specific in what you are suggesting.

This PR add the over load of the new `AnatomicalOrientation` type. It has constructors for 4 different orientation representations, enums, triplets, direction cosines... And method to convert from strings.

Reply:
Yes, `AnatomicalOrientation` type. It looks like interface issue has been resolved?